<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsWnd4Key</class>
 <widget class="QWidget" name="CSettingsWnd4Key">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>523</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="Layout_container">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>606</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="Layout_scroll_content">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="Layout_trigger_key_container">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_display_title" native="true">
            <layout class="QHBoxLayout" name="Layout_widget_title">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_group_trigger_key">
               <property name="text">
                <string>Trigger Keys</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_key" native="true">
            <layout class="QHBoxLayout" name="Layout_checkboxs">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_normal">
               <property name="text">
                <string>Normal:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_c">
               <property name="text">
                <string>Control</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_key_spacer">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_s">
               <property name="text">
                <string>Shift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_key_spacer_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_a">
               <property name="text">
                <string>Alt</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>417</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_key_res_normal">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_full" native="true">
            <layout class="QHBoxLayout" name="Layout_checkboxs_full">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_full">
               <property name="text">
                <string>Fullscreen:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_c_f">
               <property name="text">
                <string>Control</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_key_spacer_3">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_s_f">
               <property name="text">
                <string>Shift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_key_spacer_4">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_trigger_a_f">
               <property name="text">
                <string>Alt</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>417</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_key_res_fullscreen">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="tab_common">
          <attribute name="title">
           <string>Common</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="fixed_width_label_5">
               <property name="text">
                <string>File Property:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_property"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="fixed_width_label_19">
               <property name="text">
                <string>Focus (global):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_get_focus"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <item>
              <widget class="QLabel" name="fixed_width_label_26">
               <property name="text">
                <string>Reload:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_reload"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_25">
             <item>
              <widget class="QLabel" name="fixed_width_label_22">
               <property name="text">
                <string>Reveal In Explorer:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_reveal_in_explorer"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="fixed_width_label_21">
               <property name="text">
                <string>Open With Default App:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_open_with_def"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <item>
              <widget class="QLabel" name="fixed_width_label_24">
               <property name="text">
                <string>Maximize / Restore:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_maximum_restore_window"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="fixed_width_label_4">
               <property name="text">
                <string>Close:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_close">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Esc / Ctrl+W</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Folder</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="fixed_width_label_17">
               <property name="text">
                <string>Tree View:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_folder_tree_view"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="fixed_width_label_15">
               <property name="text">
                <string>Column View:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_folder_column_view"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="fixed_width_label_11">
               <property name="text">
                <string>Open Selected Item:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_folder_open_with_def"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="fixed_width_label_12">
               <property name="text">
                <string>Reveal Selected Item:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_folder_reveal"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="fixed_width_label_13">
               <property name="text">
                <string>Open Selected Item With Seer:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_folder_open_with_seer"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Image</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="fixed_width_label_8">
               <property name="text">
                <string>Rotate Left:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_img_rotate_left"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="fixed_width_label_14">
               <property name="text">
                <string>Rotate Right:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_img_rotate_right"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="fixed_width_label_6">
               <property name="text">
                <string>Actual Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_img_actual_size"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Media</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="fixed_width_label_10">
               <property name="text">
                <string>Forward:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_forward"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="fixed_width_label_16">
               <property name="text">
                <string>Backward:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_backward"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="fixed_width_label_9">
               <property name="text">
                <string>Play / Pause:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_play_pause"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="fixed_width_label_20">
               <property name="text">
                <string>Loop:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_loop"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="fixed_width_label_23">
               <property name="text">
                <string>Time Indication:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_time_indication"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="fixed_width_label_25">
               <property name="text">
                <string>Mute:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_mute"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="fixed_width_label_27">
               <property name="text">
                <string>Go To Beginning:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_media_go_start"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>PDF</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="fixed_width_label_2">
               <property name="text">
                <string>Fit Width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_pdf_fit_width"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="fixed_width_label_18">
               <property name="text">
                <string>Actual Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="keySequenceEdit_pdf_actual_size"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>204</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
